ci-operator/config/red-hat-storage/odf-console/red-hat-storage-odf-console-release-4.12.yaml

```diff
@@ -0,0 +1,79 @@
+base_images:
+  base:
+    name: release
+    namespace: openshift
+    tag: tectonic-console-builder-v23
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: tectonic-console-builder-v23
+images:
+- dockerfile_path: Dockerfile.ci
+  inputs:
+    base:
+      as:
+      - node:15
+  to: odf-console
+releases:
+  initial:
+    integration:
+      name: "4.12"
+      namespace: ocp
+  latest:
+    integration:
+      include_built_images: true
+      name: "4.12"
+      namespace: ocp
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: lint
+  steps:
+    test:
+    - as: lint
+      commands: yarn install && yarn run lint
+      from: src
+      resources:
+        requests:
+          cpu: 100m
+          memory: 200Mi
+- as: odf-console-e2e-aws
+  steps:
+    cluster_profile: aws
+    dependencies:
+      ODF_CONSOLE_FULL_IMG_NAME: odf-console
+    env:
+      COMPUTE_NODE_TYPE: m4.2xlarge
+      ZONES_COUNT: "3"
+    test:
+    - as: tests
+      cli: latest
+      commands: |
+        export PULL_SECRET="$(cat /etc/odf-console-e2e/odf-e2e-credentials/config-pull-secret)"
+        ./test-prow-e2e.sh ${ODF_CONSOLE_FULL_IMG_NAME}
+      credentials:
+      - mount_path: /var/run/operator-secret
+        name: operator-secret
+        namespace: test-credentials
+      - mount_path: /etc/odf-console-e2e/odf-e2e-credentials
+        name: odf-e2e-credentials
+        namespace: test-credentials
+      dependencies:
+      - env: ODF_CONSOLE_FULL_IMG_NAME
+        name: odf-console-img
+      from: src
+      resources:
+        requests:
+          cpu: 1000m
+          memory: 1Gi
```

Comment:
Why do we have such high resource requirement for running the cypress test?

Reply:
lets revert them.

Reply:
See https://github.com/openshift/release/pull/32612#pullrequestreview-1132449551